compiler/one-cmds/one-import-onnx

```diff
@@ -272,6 +339,22 @@ def _convert(args):
         # convert onnx to tf saved model
         onnx_model = onnx.load(getattr(args, 'input_path'))
         _sanitize_io_names(onnx_model)
+        input_shape_provided = oneutils.is_valid_attr(args, 'input_shapes')
+        output_shape_provided = oneutils.is_valid_attr(args, 'output_shapes')
+        if input_shape_provided or output_shape_provided:  # any shape arg provided
+            input_shapes_map = _extract_origin_tensor_shapes(onnx_model.graph.input)
+            if input_shape_provided:
+                input_shapes_map = _parse_shapes(
+                    getattr(args, 'input_shapes'), input_shapes_map)
+                delattr(args, 'input_shapes'
+                        )  # avoid argument colision with tf2tflite conversion step
+            output_shapes_map = _extract_origin_tensor_shapes(onnx_model.graph.output)
+            if output_shape_provided:
+                output_shapes_map = _parse_shapes(
+                    getattr(args, 'output_shapes'), output_shapes_map)
+            onnx_model = update_model_dims.update_inputs_outputs_dims(
+                onnx_model, input_shapes_map, output_shapes_map)
+            onnx_model = onnx.shape_inference.infer_shapes(onnx_model)
```

Comment:
Shouldn't we use `strict_mode` and handle errors?

https://onnx.ai/onnx/api/shape_inference.html

<img src=https://github.com/user-attachments/assets/a095b887-9102-41e8-a416-0f6e583ac53a width=800/>

Reply:
Good observation, but IHMO `infer_shapes` from `onnx` is something optional here. Even if [something goes wrong](https://github.com/onnx/onnx/blob/99e1fd352c05c3176770080824fd7a8c474c97c0/onnx/shape_inference/implementation.cc#L722C7-L722C27) we have still shape inference provided by ONE itself. My assumption was to treat it as an improvements for some edge cases ;-)

Reply:
My point is about not just dying when something goes wrong, but recognizing and dealing with the situation. If someone is running a multi-step toolchain and it crashes, make sure that the person using the toolchain knows why it crashed and what to do about it. 

Remember that not everyone using the toolchain is a brilliant programmer like you.

Reply:
How about enabling `strict_mode`, catching exception and printing warning if shapes calculated by onnx lib were not applied?

> If someone is running a multi-step toolchain and it crashes, make sure that the person using the toolchain knows why it crashed and what to do about it.

I am also OK with stopping conversion if onnx shape inference fails. You are right that I haven't a big experience with the whole related toolchain ;)

Reply:
> How about enabling `strict_mode`, catching exception and printing warning if shapes calculated by onnx lib were not applied?

Is it worth continuing execution of the toolchain after this? Is there any chance that the toolchain execution will end successfully?

If not, wouldn't it be better to display an error message explaining the problem and abort execution?

Reply:
Completely yes. ` onnx.shape_inference.infer_shapes` is not needed for models dedicated to be supported by this PR.

My proposition is to remove it now and add separately if really needed ;)

Reply:
Please proceed carefully to avoid regression. :)

Reply:
Ok, let's proceed it in this way ;)

Reply:
@lemmaa Is it ok for you now? Can I start creating PRs > ;-)

Reply:
@seanshpark Can I start introduce this feature? Is the current design acceptable for you ;-) ?